cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,16 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
```

Comment:
Can we have integration tests?

Reply:
Yes. I can add some integration tests. :)